src/simplewallet/simplewallet.cpp

```diff
@@ -2227,6 +2226,7 @@ bool simple_wallet::transfer_main(int transfer_type, const std::vector<std::stri
     else while (!ptx_vector.empty())
     {
       auto & ptx = ptx_vector.back();
+      ptx.is_onetime = is_onetime;
```

Comment:
That seems wrong. The wallet tx creation should set this if it creates such a tx. The flag is passed to it already.